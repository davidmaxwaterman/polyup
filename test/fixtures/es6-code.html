<link rel='import' href='../polymer/polymer.html'>

<polymer-element name='uses-es6'>
  <script>
    Polymer({
      created: function() {
        var world = 'the world';
        this.str = `Hello there
        ${world}
        `;
      }
    })
  </script>
</polymer-element>
