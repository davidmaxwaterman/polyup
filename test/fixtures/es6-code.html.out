<link rel="import" href="../polymer/polymer.html">

<dom-module id="uses-es6">
  <script>
    Polymer({
      is: 'uses-es6',
      created: function () {
        var world = 'the world';
        this.str = `Hello there
        ${ world }
        `;
      }
    });
  </script>
</dom-module>
